librz/core/casm.c

```diff
@@ -187,9 +187,14 @@ RZ_API RzCmdStatus rz_core_asm_plugins_print(RZ_NONNULL RZ_BORROW RzCore *core,
 	RzListIter *it;
 	RzAsmPlugin *ap;
 	RzCmdStatus status;
+	char header[128];
 	if (arch) {
 		rz_list_foreach (plugin_list, it, ap) {
 			if (ap->cpus && !strcmp(arch, ap->name)) {
+				if(state->mode == RZ_OUTPUT_MODE_STANDARD){
+					snprintf(header, sizeof(header), "CPUs for Arch[%s]:", ap->name);
+					rz_cons_println(header);
+				}
```

Comment:
`RZ_OUTPUT_MODE_STANDARD` cannot have table headers. for that there is table mode that already implements it.